<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="clients">
       <attribute name="title">
        <string>Клиенты</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <widget class="QTableView" name="clientsView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
          </widget>
          <widget class="QGroupBox" name="clientFilterGroupBox">
           <property name="title">
            <string>Фильтры</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="clientFirstNameFilter">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="clientLastNameFilter">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Паспорт</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="clientPassportFilter">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>129</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="clientClearFilterButton">
                <property name="text">
                 <string>Очистить все</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="insuranceType">
       <attribute name="title">
        <string>Виды страхования</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="insuranceTypeView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="insuranceDeal">
       <attribute name="title">
        <string>Сделки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableView" name="InsuranceDealView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Клиенты</string>
    </property>
    <addaction name="addClientAction"/>
    <addaction name="findClientAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Страховки</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Создать</string>
     </property>
     <addaction name="addInsuranceType"/>
     <addaction name="addInsuranceDeal"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Найти</string>
     </property>
     <addaction name="findInsuranceType"/>
     <addaction name="findInsuranceDeal"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="addClientAction">
   <property name="text">
    <string>Добавить клиента</string>
   </property>
  </action>
  <action name="findClientAction">
   <property name="text">
    <string>Поиск клиента</string>
   </property>
  </action>
  <action name="addInsuranceType">
   <property name="text">
    <string>Новый вид</string>
   </property>
  </action>
  <action name="addInsuranceDeal">
   <property name="text">
    <string>Новую сделку</string>
   </property>
  </action>
  <action name="findInsuranceType">
   <property name="text">
    <string>Вид</string>
   </property>
  </action>
  <action name="findInsuranceDeal">
   <property name="text">
    <string>Сделку</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clientClearFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>clientFirstNameFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clientClearFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>clientLastNameFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clientClearFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>clientPassportFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
